cmd/root.go

```diff
@@ -129,6 +131,11 @@ func parseConfig(conf *proxy.Config, args []string) error {
 		return newBadCommandError(fmt.Sprintf("not a valid IP address: %q", conf.Addr))
 	}
 
+	// If both token and credentials file were set, error.
+	if conf.Token != "" && conf.CredentialsFile != "" {
+		return newBadCommandError("ambiguous auth configuration: both token and credentials file were set")
```

Comment:
nit: suggest wording like "Cannot specify `--token` and `--credentials-file` flags at the same time". 

There are likely going to be a few more cases of conflicting flags, we should just be consistent